lib/tasks/check_create_es_data_from_pg.rake

```diff
@@ -0,0 +1,27 @@
+namespace :check do
+	# bundle exec rake check:create_es_data_from_pg
+	# will create ES from PG
+  desc "Create ES data from PG"
+  task create_es_data_from_pg: :environment do
+  	MediaSearch.delete_index
+  	MediaSearch.create_index
+  	# Add ES parent
+  	[ProjectMedia, ProjectSource].each do |type|
+  		type.find_each do |obj|
+  			obj.add_elasticsearch_data
+  			print '.'
+  		end
+  	end
+  	sleep 10
+  	# Add ES for annotations (child items)
+  	Annotation.find_each do |a|
+  		a = a.load
+      klass = a.class.name.downcase
+      if ['comment', 'tag', 'dynamic'].include?(klass)
```

Comment:
can we use `a.respond_to("add_update_elasticsearch_#{klass}")` instead of hard-coding the types of the annotations?

Reply:
Sure